packages/@coorpacademy-nova-icons/icons.js

```diff
@@ -245,7 +245,11 @@ const icons: Array<Icon> = [
   {filePath: path.resolve('./third-party/nova-solid.iconjar/icons/sd-card.svg')},
   {filePath: path.resolve('./third-party/nova-solid.iconjar/icons/folders.svg')},
   {filePath: path.resolve('./third-party/nova-solid.iconjar/icons/file-block-2.svg')},
-  {filePath: path.resolve('./third-party/nova-line.iconjar/icons/cookie.svg')}
+  {filePath: path.resolve('./third-party/nova-line.iconjar/icons/cookie.svg')},
+  // review
```

Comment:
Comment to delete

Reply:
this is to separate the review icons from others you can see upper in the file that there is already some others to separate like podcast and so

Reply:
<img width="960" alt="image" src="https://user-images.githubusercontent.com/22681512/208877638-cd1a0286-3745-4c08-bb7c-844bc0e68875.png">

Reply:
you are right !